README.md

```diff
@@ -1,3 +1,4 @@
+
 Clicking [this link](ALBUM.md) will take you to the album page.
 
 My name is Emmanuel Kusi a young talented footballer from Ghana in the West Africa. I have a dream and i love my dream, I want to become a professional footballer and travel around europe. I want to become one of the greatest footballers who has ever kicked the ball. I want to win trophies with my country and any club teams i join,I train hard and train my mind very well and am looking forward to getting an opportunity to make my dream come true. People plays football because of the money, others play because of the fame and name,but I play football because i love football,I love the game so much.
```

Comment:
"but I play" and "I love the game" have no space before them and after the comma, commas must have spaces after them always